Invite user instructions (#4511)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/invite-users.md

---
title: "Invite users to dbt Cloud"
description: "Learn how to manually invite users to dbt Cloud"
id: "invite-users"
sidebar: "Invite users"
---

dbt Cloud makes it easy to invite new users to your environment out of the box. This feature is available to all dbt Cloud customers on Teams or Enteprise plans (Developer plans are limited to a single user).

> ## Invite new users

Reviewer: sugg turning this into prerequs

```suggestion
## Prerequisites
```

You must have proper permissions to invite new users:

- [**Teams accounts**](/docs/cloud/manage-access/self-service-permissions) &mdash; must have `member` or `owner` permissions.
- [**Enterprise accounts**](/docs/cloud/manage-access/enterprise-permissions) &mdash; must have `admin`, `account admin`, `project creator`, or `security admin` permissions.

> 

Reviewer: they ned to have a developer license seat too, right? i think we should add that here because I've had users reach out and are read only seatsand owner permissions reach out wondering why they can't invite users (or manage acct settings)

> To invite users: 

Reviewer: sugg turning to h2
```suggestion
## Invite new users
```

> 1. Open the **Account Settings** from the gear menu in your dbt Cloud account.

Reviewer: ```suggestion
1. In your dbt Cloud account, select the gear menu in the upper right corner and then select **Account Settings**.
```

2. From the left sidebar, select **Users**.

> <Lightbox src="/img/docs/dbt-cloud/access-control/users-screen.png" title="The user screen in settings"/>

Reviewer: ```suggestion
<Lightbox src="/img/docs/dbt-cloud/access-control/users-screen.png" width"70%" title="The user screen in the Account Settings page"/>
```

3. Click on **Invite Users**.

<Lightbox src="/img/docs/dbt-cloud/access-control/invite-users.png" title="The invite users pane"/>

4. In the **Email Addresses** field, enter the email addresses of the users you would like to invite separated by comma, semicolon, or a new line.
5. Select the license type for the batch of users from the **License** dropdown.
6. Select the group(s) you would like the invitees to belong to.
7. Click **Send Invitations**.

>     - If the list of invitees exceeds the number of licenses your account has availabe, you will receive a warning when you click **Send Invitations** and the invitations will not be sent. 

Reviewer: ```suggestion
    - If the list of invitees exceeds the number of licenses your account has available, you will receive a warning when you click **Send Invitations** and the invitations will not be sent. 
```

>     - If the list of invitees exceeds the number of licenses your account has availabe, you will receive a warning when you click **Send Invitations** and the invitations will not be sent. 

Reviewer: maybe this section can be its own non bulleted line and we can also have subtle guidance for users that exceed their seats and would like to increase that. e.g.:

> If the list of invitees exceeds the number of licenses your account has available, you will receive a warning when you click **Send Invitations** and the invitations will not be sent. Need more licenses? Check out the [pricing plan options](https://www.getdbt.com/pricing) to upgrade your account.

> 

Reviewer: might be good to add a blurb that they can use the same flow to reinvite users

Author: Added to faqs

## User experience

> dbt generates and sends the email to the addresses defined in dbt Cloud. Be sure to allow traffic from `support@getdbt.com` to prevent emails from going to spam or being blocked (this is the originating email address for all [instances worldwide](/docs/cloud/about-cloud/regions-ip-addresses)).

Reviewer: is it 'dbt' or 'dbt Cloud generates and sends..
```suggestion
dbt Cloud generates and sends emails from `support@getdbt.com` to the specified addresses. Make sure traffic from the `support@getdbt.com` email is allowed in your settings to avoid emails from going to spam or being blocked. This is the originating email address for all [instances worldwide](/docs/cloud/about-cloud/regions-ip-addresses).

```

Author: Do we want to be this general? Do all emails the platform generates come from the support email address? If not, I think we should specify here that this is only for this workflow.

Reviewer: hm not sure tbh (e.g. verify emails for cloud). but good flag to specify. maybe:

>dbt Cloud generates and sends invitation emails from `support@getdbt.com` to the specified addresses. Make sure traffic from the `support@getdbt.com` email is allowed in your settings to avoid emails from going to spam or being blocked. This is the originating email address for all [instances worldwide](/docs/cloud/about-cloud/regions-ip-addresses).

> The email contains a link to create an account. When the user clicks on this they will be brought to one of two screens depending on whether SSO is configured or not.

Reviewer: maybe the two diff flows can present in tabs so it's clearer and users can go directly to their use case?

> <Lightbox src="/img/docs/dbt-cloud/access-control/email-invite.png" title="Example or an email invitation"/>

Reviewer: i might be wrong but i thought brand new users needed to verify their email address? if that's still the flow, might be good to add that?

Author: This is not a step I encountered through multiple tests. After the link, you create the account, and it takes you directly to the portal

The default settings send the email, the user clicks the link, and is prompted to create their account:

<Lightbox src="/img/docs/dbt-cloud/access-control/default-user-invite.png" title="Default user invitation"/>

If SSO is configured for the environment, the user clicks the link, is brought to a confirmation screen, and presented with a link to authenticate against the company's identity provider:

<Lightbox src="/img/docs/dbt-cloud/access-control/sso-user-invite.png" title="User invitation with SSO configured"/>

Once the user completes this process, their email and user information will populate in the **Users** screen in dbt Cloud.

## FAQ

> * Is there a limit to the number of users I can invite? _Your ability to invite users is limited to the number of licenses you have available._

Reviewer: might be good to add something about if the email has a typo? what do they do if the email they invited is wrong? who do they reach out to?

Author: Do they need to reach out to someone? Wouldn't they send another invite? Is there something support needs to do?

Reviewer: i dont think they reach out to support (although i def do remember users reaching out to support when they had typos,etc). but i believe they would need to create an account with the correct email. @BrJan can confirm though! [also this may have insight](https://www.notion.so/dbtlabs/How-to-assist-a-customer-that-registered-with-the-incorrect-email-address-8f08017f58d84983841c0df5dfc25dd6?pvs=4)

You must have proper permissions to invite new users:

- [**Teams accounts**](/docs/cloud/manage-access/self-service-permissions) &mdash; must have `member` or `owner` permissions.
- [**Enterprise accounts**](/docs/cloud/manage-access/enterprise-permissions) &mdash; must have `admin`, `account admin`, `project creator`, or `security admin` permissions.
- The admin inviting the users must have a `developer` or `IT` license.

## Invite new users

1. In your dbt Cloud account, select the gear menu in the upper right corner and then select **Account Settings**.
2. From the left sidebar, select **Users**.

<Lightbox src="/img/docs/dbt-cloud/access-control/users-screen.png" title="The user screen in the Account Settings page"/>

3. Click on **Invite Users**.

<Lightbox src="/img/docs/dbt-cloud/access-control/invite-users.png" title="The invite users pane"/>

4. In the **Email Addresses** field, enter the email addresses of the users you would like to invite separated by comma, semicolon, or a new line.
5. Select the license type for the batch of users from the **License** dropdown.
6. Select the group(s) you would like the invitees to belong to.
7. Click **Send Invitations**.

>     - If the list of invitees exceeds the number of licenses your account has available, you will receive a warning when you click **Send Invitations** and the invitations will not be sent. 

Reviewer: i do wonder if we should have info on how to add more seats or upgrade.
```suggestion
If the list of invitees exceeds the number of licenses your account has available, you will receive a warning when you click **Send Invitations** and the invitations will not be sent. Need more licenses? Check out the [pricing plan options](https://www.getdbt.com/pricing) to upgrade your account.
```

## User experience

dbt Cloud generates and sends emails from `support@getdbt.com` to the specified addresses. Make sure traffic from the `support@getdbt.com` email is allowed in your settings to avoid emails from going to spam or being blocked. This is the originating email address for all [instances worldwide](/docs/cloud/about-cloud/regions-ip-addresses).

The email contains a link to create an account. When the user clicks on this they will be brought to one of two screens depending on whether SSO is configured or not.

<Lightbox src="/img/docs/dbt-cloud/access-control/email-invite.png" title="Example or an email invitation"/>

<Tabs>

> <TabItem value="Local user">

Reviewer: what is. a local user?
